app/src/ui/history/commit-list.tsx

```diff
@@ -377,11 +377,18 @@ export class CommitList extends React.Component<ICommitListProps, {}> {
     const { commitSHAs, selectedSHAs, shasToHighlight, emptyListMessage } =
       this.props
     if (commitSHAs.length === 0) {
-      return <div className="panel blankslate">{emptyListMessage}</div>
+      return (
+        <div className="panel blankslate">
+          {emptyListMessage === undefined
+            ? 'No commits to list'
+            : emptyListMessage}
```

Comment:
Maybe we can use this instead? (Warning: this suggestion might require linting)
```suggestion
          {emptyListMessage ?? 'No commits to list'}
```